test/java/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java-library'
+
+repositories {
+    mavenLocal()
+    jcenter()
+}
+
+task generateSysl(type:Exec) {
+  commandLine 'reljam',  '--out', 'src/main/java', 'model', '/tuplecomplex', 'UserFormComplex'
+
+  standardOutput = new ByteArrayOutputStream()
+
+  ext.output = {
+    return standardOutput.toString()
+  }
+}
+
+compileJava.dependsOn generateSysl
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ["src/main/java", "../../src/libs/java"]
+        }
+        resources {
+            srcDir 'src/main/resources'
+        }
+    }
+
+    test {
+        java {
+            srcDirs = ["src/test/java"]
```

Comment:
move to `srcDirs = ["src"]`

Reply:
This is still not done.

Reply:
["src/main/java", "../../src/libs/java", "$tempDirectory"]

what is wrong with it? 
My understanding is 
java has src/{main,test}/.. directories...